static/scss/answers/collapsible-filters/collapsible-filters-templates.scss

```diff
@@ -45,6 +45,26 @@
 .AnswersVerticalGrid.CollapsibleFilters {
   @include CollapsibleFiltersOverrides;
 
+  .Answers-resultsHeader, .Answers-results {
+    @media (min-width: #{$breakpoint-expanded-filters}) {
+      width: $answers-container-width-percentage;
+    }
+    width: var(--hh-answers-container-width);
+    max-width: var(--hh-answers-container-width);
+  }
+
+  .Answers-resultsHeader {
+    @media (min-width: #{$breakpoint-expanded-filters}) {
+      margin: 0 math.div(100% - $answers-container-width-percentage,2);
```

Comment:
This does improve the issue, however the header will still snap right above 3233px which could happen while fullscreen on a 4k monitor. The issue is that this calculated percentage doesn't take into account the results having a max width

While we could possibly resolve this issue, I feel that we should probably hold of on this PR because of the complexity that this change would introduce